packages/astroport/src/vesting.rs

```diff
@@ -57,6 +57,11 @@ pub enum ExecuteMsg {
     /// ## Executor
     /// Only the current owner can execute this
     RemoveVestingManagers { managers: Vec<String> },
+    ///
```

Comment:
please describe the message and its permissions. stick with the pattern used in other messages

Reply:
actually I meant something like this:

```rust
    /// Sets the vesting token
    /// ## Executor
    /// Only the current owner or token info manager can execute this
    SetVestingToken {
        /// [`AssetInfo`] of the token that's being vested
        vesting_token: AssetInfo,
    },
```